deploy/charts/kaudit/values.yaml

```diff
@@ -83,13 +83,15 @@ aws:
 
 k8s:
   # webhook or auditsink
-  mode: "auditsink" 
+  mode: "auditsink"
+  # if k8s cluster is running via openshift set flag to "true"
+  openshift: "false"
```

Comment:
This can be removed once you have the other changes implemented

Reply:
removed